Signal/src/view controllers/CallViewController.swift

```diff
@@ -519,7 +519,6 @@ class CallViewController: UIViewController, CallDelegate {
 
     func updateCallUI(callState: CallState) {
         let textForState = localizedTextForCallState(callState)
-        Logger.info("\(TAG) new call status: \(callState) aka \"\(textForState)\"")
```

Comment:
Good point. State change is already logged in the SignalCall, so leaving out the call in updateCallUI.